src/packages/end-to-end/src/__tests__/api/mutation/basic-create.test.ts

```diff
@@ -19,7 +19,7 @@ describe('basic mutation', () => {
 					}
 				`
 			)
-			.variables({ data: { ArtistId: { id: 1 }, Title: 'string' } })
```

Comment:
Looks like we followed through on changing the db naming convention as discussed in that meeting with Kevin.

Reply:
yeah correct